.github/workflows/build-test.yml

```diff
@@ -12,10 +12,19 @@ jobs:
         run: ./build-rootless.sh $(echo $GITHUB_REPOSITORY |cut -d '/' -f1)
       - name: Build standard image
         run: ./build.sh $(echo $GITHUB_REPOSITORY |cut -d '/' -f1)
-      - name: Run tests
-        run: cd tests; ./run_tests.sh
       - name: List Docker images
         run: docker images --filter "reference=puppet-dev-tools*" --filter "reference=*/puppet-dev-tools*"
       - name: Show Docker image labels
         run: |
           docker inspect --format='{{json .Config.Labels}}' ${{ secrets.DOCKERHUB_USERNAME }}/puppet-dev-tools
+      - name: Run tests
+        run: cd tests; ./run_tests.sh
```

Comment:
I am not familiar with this pipeline, but...

How come you re-ordered the steps?

Also that comment at the top about "should be identical" makes me wonder if there was a reason they were in a particular order? Does `run_tests.sh` produce new images and we wanted to list them out after the tests? 🤷‍♂️ 

Also no big deal but GHA has the following for setting the CWD:
```
- name: Run tests
  working-directory: tests
  run: ./run_tests.sh
```

Or to be explicit:
```
working-directory: ${{ github.workspace }}/tests
```

Reply:
I re-ordered them because I wanted the list step to be close to the step that actually builds the images (`run-tests` does not build any new ones). In the pipeline this is based on, there was a tagging step between `run-tests` and the `list` step, which is not included in this pipeline since we're not pushing anything. So there it makes sense to do the list after the tests _and_ the tagging, but not here IMO. I might actually remove that comment, I'm not sure it's helpful.

I'll make that update for the working dir, thanks!